content/doc/developer/plugin-development/writing-an-scm-plugin.adoc

```diff
@@ -31,12 +31,12 @@ references:
 
 == Introduction
 
-This document is a howto on writing an SCM plugin for Jenkins.
+This document is a how-to on writing an SCM plugin for Jenkins.
 It is based on my experiences writing the link:https://plugins.jenkins.io/tfs[TFS Plugin].
 
 == Prerequisites
 
-For the rest of this howto, it is assumed that the plugin development environment has been set up according to the .
+For the rest of this how-to, it is assumed that the plugin development environment has been set up according to the link:https://jenkins.io/doc/developer/tutorial/[Plugin tutorial].
```

Comment:
```suggestion
For the rest of this how-to, it is assumed that the plugin development environment has been set up according to the link:/doc/developer/tutorial/[Plugin tutorial].
```

Reply:
not really resolved

Reply:
sorry, I‘m not familiar with this workflow before.